From 16947bd68552e7db8ececf87cc1b5b10ac5c59eb Mon Sep 17 00:00:00 2001
From: Christophe Continente <christophe.continente@intel.com>
Date: Thu, 6 Sep 2012 12:04:26 +0200
Subject: [PATCH 01/22] [PORT FROM MAIN] build target update for the
 ctp_nomodem variant

BZ: 55504

Call the telephony.mk only if the modem is validated into the board.mk platform file.

[BOOT CAMP SPECIFIC]

Change-Id: Ibbe0aa828a463e8f9a15a0aba8ff93132b0d2d36
Signed-off-by: Christophe Continente <christophe.continente@intel.com>
Reviewed-on: http://android.intel.com:8080/64649
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/74480
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 target/product/full_base_no_telephony.mk |   29 +++++++++++++++++++++++++++++
 1 files changed, 29 insertions(+), 0 deletions(-)
 create mode 100644 target/product/full_base_no_telephony.mk

diff --git a/target/product/full_base_no_telephony.mk b/target/product/full_base_no_telephony.mk
new file mode 100644
index 0000000..9e57e48
--- /dev/null
+++ b/target/product/full_base_no_telephony.mk
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2009 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is a build configuration for a full-featured build of the
+# Open-Source part of the tree. It's geared toward a US-centric
+# build quite specifically for the emulator, and might not be
+# entirely appropriate to inherit from for on-device configurations.
+
+PRODUCT_PACKAGES := \
+    VoiceDialer
+
+PRODUCT_PROPERTY_OVERRIDES := \
+    keyguard.no_require_sim=true \
+    ro.com.android.dataroaming=true
+
+$(call inherit-product, $(SRC_TARGET_DIR)/product/full_base.mk)
-- 
1.7.9

